tasks/modules/logging/collins.rb

```diff
@@ -0,0 +1,18 @@
+module Logging
+  module Collins
+
+    def self.collins
+      Genesis::Framework::Utils.collins
+    end
+
+    def self.facter
+      Genesis::Framework::Utils.facter
+    end
+
+    def self.log message
+      if self.facter['asset_tag']
+        collins.log! 'tumblrtag301', message
```

Comment:
Whoops, this is from me testing it out. One of the assets added by the collins script that populates the db with example data has the tag `tumblrtag301` :)

This should be `facter['asset_tag']` instead. Fixing.